cerberus/validator.py

```diff
@@ -97,6 +97,10 @@ class and a dictionary, the latter is passed to the
     """ Rules that will be processed in that order before any other and abort
         validation of a document's field if return ``True``.
         Type: :class:`tuple` """
+    post_priority_validation = ('dependencies',)
+    """ Rules that will be processed if defined when one of the priority 
```

Comment:
i do not exactly understand that sentence. in particular the 'if' followed by 'when' without a conjunction of the condition expressions (if they are that, i'm assuming).

`flake8` tells us, there's a trailing whitespace here.